src/app/ngrx/effects/cms.effects.spec.ts

```diff
@@ -1,124 +1,208 @@
-      {id: 123, title: 'A Pet Talk Episode', publishedAt: new Date('2018-01-01')},
-      {id: 124, title: 'A More Recent Pet Talk Episode', publishedAt: new Date('2018-01-02')}
-    ]);
-    stories[0].mockItems('prx:distributions',
-        [{kind: 'episode', url: 'https://feeder.prx.org/api/v1/episodes/abcdefg'}]);
-    stories[1].mockItems('prx:distributions',
-      [{kind: 'episode', url: 'https://feeder.prx.org/api/v1/episodes/gfedcba'}]);
-    episodes[0].doc = stories[0];
-    episodes[1].doc = stories[1];
+    auth = cms.mock('prx:authorization', {});
+  });
 
+  beforeEach(() => {
     TestBed.configureTestingModule({
-      declarations: [
-        DownloadsComponent,
-        DownloadsChartComponent,
-        DownloadsTableComponent
-      ],
       imports: [
         RouterTestingModule,
-        downloadsRouting,
-        AuthModule,
-        FancyFormModule,
-        SharedModule,
-        StoreModule.forRoot(reducers),
-        EffectsModule.forRoot([CmsEffects])
+        StoreModule.forRoot(reducers)
       ],
       providers: [
+        {provide: AuthService, useValue: {token: authToken}},
+        {provide: HalService, useValue: cms},
+        CmsService,
         CmsEffects,
-        {provide: CmsService, useValue: cms.root},
-        { provide: Actions, useFactory: getActions }
+        provideMockActions(() => actions$)
       ]
     });
-
     effects = TestBed.get(CmsEffects);
-    actions$ = TestBed.get(Actions);
-
-    spyOn(effects, 'routeWithEpisodeCharted').and.callThrough();
-
-    effects.routeParams = {
-      beginDate: '2017-12-03T00:00:00.000Z',
-      chartPodcast: 'true',
-      endDate: '2017-12-14T23:59:59.999Z',
-      episodes: '',
-      interval: 'daily',
-      page: '1',
-      seriesId: '37800',
-      standardRange: '2 weeks'
-    };
-
-    const action = new CmsPodcastEpisodePageAction({podcast, page: 1});
-    effects.store.dispatch(action);
-    actions$.stream = hot('-a', { a: action });
-    const result = new CmsPodcastEpisodePageSuccessAction({episodes});
-    expected = cold('-r', { r: result });
-  }));
-
-  it('should create load episodes page and dispatch CmsPodcastEpisodePageSuccessAction with page of episodes', () => {
-    expect(effects.loadEpisodes$).toBeObservable(expected);
+    spyOn(effects, 'routeWithEpisodeCharted').and.stub();
+    spyOn(effects, 'getEpisodeColor').and.returnValue('#fff');
+  });
+
+  describe('loadAccount', () => {
+
+    it('successfully loads the account', () => {
+      const accounts = auth.mockItems('prx:accounts', [
+        {name: 'TheAccountName', type: 'IndividualAccount', id: 111},
+        {name: 'DefaultName', type: 'DefaultAccount', id: 222}
+      ]);
+      const account = {id: 111, name: 'TheAccountName', doc: accounts[0]};
+      const action = new ACTIONS.CmsAccountAction();
+      const completion = new ACTIONS.CmsAccountSuccessAction({account});
+      actions$ = hot('-a', {a: action});
```

Comment:
I have no idea if i'm doing these marbles correctly or not.

Reply:
looks right to me, expect to get an observable of the success action with that account payload